interface/documents/upload.php

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
I don't know how to clear out the git so that it only uploads what I am currently working on instead of uploading my back log. Any guidance?